src/platforms/dotnet/diagnostic-logger.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Troubleshooting and diagnosing issues
+description: "How to enable logging in Sentry SDK for troubleshooting purposes."
+sidebar_order: 20
+---
+
+Sentry has its own logger that it uses to report diagnostics which may be useful when troubleshooting. To enable logging, set the `Debug` option to `true`:
+
+```csharp
+SentrySdk.Init(o =>
+{
+    o.Dsn = "___PUBLIC_DSN___";
+
+    // Enable debug mode to write diagnostic messages
+    o.Debug = true;
+});
+```
+
+# Change the diagnostic logger
+
+By default, Sentry will write diagnostic messages to console. This may not be optimal in some circumstances, for example when running applications that don't have a visible console window attached.
```

Comment:
```suggestion
By default, Sentry will write diagnostic messages to console. This may not be optimal in some circumstances; for example, when running applications that don't have a visible console window attached.
```